src/App.test.tsx

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import App from './App';
 
```

Comment:
Leave only one empty line before last inport line and following code. Do it for all files on this pull request (PR)